src/script/editor/tool/index.js

```diff
@@ -34,6 +33,9 @@ import ReactionUnmapTool from './reactionunmap';
 import PasteTool from './paste';
 import RotateTool from './rotate';
 
+// TODO: correct
+import EnhancedStereoTool from './enhanced-stereo';
```

Comment:
Looks like tool working, remove TODO and empty lines

Reply:
Done